app/views/taxonomies/_form.html.erb

```diff
@@ -99,7 +99,18 @@
 
       <% if User.current.allowed_to?(:view_params) %>
         <div class="tab-pane" id="params">
-          <%= render "common_parameters/parameters", { :f => f, :type => "#{controller_name.singularize}_parameters".to_sym } %>
+          <% taxonomy_type = controller_name.singularize %>
+          <% if taxonomy.parent.present? %>
+              <h4><%= _('Parent parameters') %></h4>
+              <%= render "common_parameters/inherited_parameters", { :inherited_parameters => taxonomy.parent_params(true),
+                                                                     :parameters => taxonomy.send("#{taxonomy_type}_parameters".to_sym)} %>
+          <% end %>
+          <% if taxonomy.is_a?(Location) %>
+              <h4><%= _("Location parameters") %></h4>
```

Comment:
It wouldn't, and while you could translate the words separately, I think this should stay as-is so the translators have the full context/sentence.